src/components/ProductList/ProductList.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactNode } from "react";
+
+interface ProductListProps {
+  children: ReactNode;
+}
+
+export const ProductList = ({ children }: ProductListProps): ReactNode => {
```

Comment:
No need to declare the return type, it can be inferred by typescript (if you hover over `ProductList`, even after deleting it, you will see it is specified correctly).

For type declarations, the rule we follow is: if typescript can infer it, then don't type it.

Reply:
Thank you!